packages/@aws-cdk/aws-lambda-nodejs/test/bundling.test.ts

```diff
@@ -376,3 +376,24 @@ test('with command hooks', () => {
     }),
   });
 });
+
+test('esbuild bundling with projectRoot', () => {
```

Comment:
We also need a test with dependencies (`nodeModules`).

Reply:
See question above

Reply:
Any updates?

Reply:
https://github.com/aws/aws-cdk/pull/15055#discussion_r660388591 needs to be fixed and then a test must be added